vispy/plot/plot.py

```diff
@@ -18,7 +19,10 @@ def plot(*args, **kwds):
     canvas.view = canvas.central_widget.add_view()
     canvas.line = visuals.LinePlot(*args, **kwds)
     canvas.view.add(canvas.line)
-    canvas.view.camera.auto_zoom(canvas.line)
+    if False:  # todo: of data-is-3D
```

Comment:
I think we might at some point place objects at nonzero z positions to achieve z ordering. E.g. move images slightly to the back so that lines/markers drawn on top are visible. In visvis I push existing images a bit backward each time `imshow()` gets called, so that the last image is always fully visible.

My proposal is that inside the "plot" functions here, we set the camera based on the given data. We could also check if the camera is already changed (from being the BaseCamera), and not set the camera in that case.